components/edit-collective/sections/virtual-cards/VirtualCards.js

```diff
@@ -170,7 +185,16 @@ const VirtualCards = props => {
       </Box>
       <Grid mt={4} gridTemplateColumns={['100%', '366px 366px']} gridGap="32px 24px">
         {data.account.virtualCards.nodes.map(vc => (
-          <VirtualCard key={vc.id} {...vc} />
+          <VirtualCard
+            canEditVirtualCard={
+              props.collective.type === CollectiveType.USER &&
+              vc.provider === 'STRIPE' &&
+              (vc.data.state === 'OPEN' || vc.data.status === 'active')
+            }
```

Comment:
What happens if I go to my **Collective** settings, Virtual Cards? Will I not be able to pause the card from there?

This should have been the first thing to do, before even creating a new Virtual Cards section for Individuals.

Reply:
I've refactored the code so that we have the ability to pause cards from the collective settings -> virtual cards section. And removed the Virtual Cards section for individuals.